Services/Administration/OJS.Services.Administration.Business/Implementations/ProblemGroupsBusinessService.cs

```diff
@@ -89,7 +89,10 @@ public async Task ReevaluateProblemsAndProblemGroupsOrder(int contestId, Problem
             await this.problemGroupsOrderableService.ReevaluateOrder(problemGroups);
 
             // We detach the existing entity, in order to avoid tracking exception on Update.
+            // if (problemGroup != null)
+            // {
```

Comment:
Delete

Reply:
It was a forgotten comment. I uncommented it because we need the logic.